diktat-gradle-plugin/src/test/kotlin/com/saveourtool/diktat/plugin/gradle/DiktatGradlePluginTest.kt

```diff
@@ -37,13 +39,4 @@ class DiktatGradlePluginTest {
         Assertions.assertIterableEquals(project.fileTree("src").files, actualInputs.files)
         Assertions.assertTrue(actualInputs.files.isNotEmpty())
     }
-
-    @Test
```

Comment:
why removed?

Reply:
this logic is not valid anymore -- we don't have such property and such method.
But similar behavior is tested in `DiktatJavaExecTaskTest` -- we expect `PlainReporter` there when no reporter is configured